hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -1695,26 +1707,26 @@ def test_not_equal(x, y):
             key=lambda i: sort_key(st.nodes[spans[i].start : spans[i].end]),
         )
 
-    def run_node_program(self, i, description, original, repeats=1):
+    def run_node_program(self, i, program, original, repeats=1):
         """Node programs are a mini-DSL for node rewriting, defined as a sequence
         of commands that can be run at some index into the nodes
 
         Commands are:
 
             * "X", delete this node
 
```

Comment:
I wonder if we should refactor this method, since the only kind of program we support is "n deletions"?